src/components/JsonField.tsx

```diff
@@ -61,7 +71,7 @@ export const JsonField: FC<FieldProps<JsonFieldSchema>> = ({
           const getPlaceholder = () => {
             if (value) {
               try {
-                return JSON.parse(value);
+                return isString(value) ? JSON.parse(value) : value;
```

Comment:
JSON.parse can throw, it it possible to move this to a function where the error is caught and returns {}